packages/clerk-js/src/utils/captcha/getCaptchaToken.ts

```diff
@@ -0,0 +1,26 @@
+import type { CaptchaProvider, CaptchaWidgetType } from '@clerk/types';
+
+import { getHCaptchaToken } from './hcaptcha';
+import { getTunstileToken } from './turnstile';
+
+type CaptchaOptions = {
+  siteKey: string;
+  scriptUrl: string;
+  widgetType: CaptchaWidgetType;
+  invisibleSiteKey: string;
+  captchaProvider: CaptchaProvider;
+};
+
+export const getCaptchaToken = async (captchaOptions: CaptchaOptions) => {
+  const { captchaProvider, widgetType, invisibleSiteKey, siteKey, scriptUrl } = captchaOptions;
+  if (captchaProvider === 'hcaptcha') {
+    return await getHCaptchaToken({ siteKey, scriptUrl, widgetType, invisibleSiteKey });
+  } else {
+    return await getTunstileToken({
```

Comment:
🙃 there is no need for `await` since there is no error handling or something in which we would handle the promise resolution here.